### DISCUSSION TITLE
Non responsive bot

### DISCUSSION THREAD

**Discussion by Anoirrz** [jagrosh/MusicBot#612] [Questions]
Hi jagrosh 

I just added your bot by following the steps in the guide, but for some reason it doesnt not respond to any of my commands.
Do you have any idea why this is ?
I already installed the lastest java and also saved the file in documents as mentioned.. 
I configured the cfg file and added the bot the my server to test it out. but i just dont react to any command

**Comment by MichailiK** [comment]
There could be many, many reasons. For example, is the bot online? Can it read & send messages? *Try giving it Administrator permission to see if it's a permissions issue.*
